### PR TITLE
Accept invite flow

### PR DESCRIPTION
**Pull request by lukachi**

### CODE REVIEW

File: src/api/modules/auth/types/index.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export type AuthTokens = {
+  id: string
+  type: 'token'
+  accessToken: {
+    token: string
+    tokenType: 'access'
+  }
+  refreshToken: {
+    token: string
+    tokenType: 'access'
```
Comment (ardier16) [inline src/api/modules/auth/types/index.ts:10]: typo on backend?

File: src/api/modules/zkp/types/index.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      }
+    }
+  }
+}
+
+export type CredentialSubject = Record<string, string> & {
+  metadata_id: string
+}
+
+export type { SaveCredentialsRequestParams, ZKProof } from '@rarimo/rarime-connector'
```
Comment (ardier16) [inline src/api/modules/zkp/types/index.ts:96]: what's the reason of the re-export?

File: src/api/modules/orgs/helpers/org-groups.ts [hunk 3]
```diff
@@ -50,6 +50,9 @@ const DUMMY_ORG_GROUP: OrgGroup[] = [
   },
 ]
 
+// DUMMY_INVITE_LINK
+// http://localhost:8095/i?q=eyJvcmdfaWQiOiJlZmM3MjhmMC0zMzEwLTQxOWEtODdlZi01ZTVlNDAyOGI4YzAiLCJncm91cF9pZCI6IjU0NGQxNmY0LWQ4YzMtNDYyYy05NjIxLWVjYzg3OTQ5NGQ0MSIsImludml0ZV9lbWFpbF9pZCI6IjVjMmFmZWIwLWRmYjYtNDdiNS1iNmE1LWQ1NmIzOTVkNmU5MyIsIm90cCI6IjEyMzQ1NiJ9
```
Comment (ardier16) [inline src/api/modules/orgs/helpers/org-groups.ts:54]: why it's here? mb move it to EmailVerification component where it's parsed?

File: src/store/modules/auth.module.ts [hunk 4]
```diff
@@ -1,17 +1,31 @@
...
 const [authStore, useAuthState] = createStore(
   'auth',
   {
-    jwt: '',
+    tokens: [],
   } as AuthState,
   state => ({
-    setJwt: (jwt: string) => {
-      state.jwt = jwt
+    addToken: (tokens: AuthTokens, orgId: string, groupId: string) => {
```
Comment (ardier16) [inline src/store/modules/auth.module.ts:19]: addTokens, not token?

File: src/enums/routes.ts [hunk 5]
```diff
@@ -20,4 +20,8 @@ export enum RoutePaths {
   OrgsIdGroupsNew = '/organisations/:id/groups/new',
 
   VerifyProofAlias = '/v/:id',
+
+  AcceptInvite = '/i',
```
Comment (ardier16) [inline src/enums/routes.ts:24]: invitation, not invite

File: src/enums/routes.ts [hunk 6]
```diff
@@ -20,4 +20,8 @@ export enum RoutePaths {
...
+  AcceptInviteEmailVerification = '/i/verify-email',
+  AcceptInviteFulfillRequest = '/i/fulfill-request',
```
Comment (ardier16) [inline src/enums/routes.ts:26]: fulfill or fill?

File: src/pages/AcceptInvite/pages/EmailVerification.tsx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    return createdRequest
+  }, [authorize, inviteDetails.group_id, inviteDetails.org_id, inviteDetails.otp, userDid])
+
+  const {
+    data: createdRequest,
+    isLoading,
+    isLoadingError,
+  } = useLoading(null, acceptInvite, {
+    loadOnMount: true,
+    loadArgs: [inviteDetails],
```
Comment (ardier16) [inline src/pages/AcceptInvite/pages/EmailVerification.tsx:54]: invite -> invitation everywhere

File: src/pages/AcceptInvite/pages/EmailVerification.tsx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+  }>(() => {
+    try {
+      // TODO: add searchParameter const
+      return JSON.parse(atob(searchParams.get('q') || ''))
+    } catch (error) {
+      return null
+    }
+  }, [searchParams])
+
+  const acceptInvite = useCallback(async () => {
```
Comment (ardier16) [inline src/pages/AcceptInvite/pages/EmailVerification.tsx:29]: this func does not only accept invitation, it does more

File: src/api/modules/auth/helpers/authorize.ts [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+  zkProof: ZKProof
+}) => {
+  const { data } = await api.post<AuthTokens>('/v1/authorize', {
+    body: {
+      data: {
+        id: userDid,
+        type: 'authorize',
+        attributes: {
+          proof: {
+            role: OrgUserRoles.Undefined, // FIXME: should be in background
```
Comment (ardier16) [inline src/api/modules/auth/helpers/authorize.ts]: do we need FIXME comment here?

**Review by ardier16 — CHANGES_REQUESTED** [review]

File: src/store/modules/auth.module.ts (hunk 4 above)
Comment (lukachi) [reply src/store/modules/auth.module.ts:19]: could we define:

app tokens - is a "token"
"authTokens" - is an group of access + refresh tokens?

**Pull request merged**